### PR TITLE
example: add url-loader for iconfont

### PR DESCRIPTION
**Pull request by noscripter**
- [ ] This is a **bugfix**
- [ ] This is a **feature**
- [ ] This is a **code refactor**
- [ ] This is a **test update**
- [x] This is a **docs update**
- [ ] This is a **metadata update**

### Motivation / Use-Case

example: add example to show usage of url-loader for iconfont

### CODE REVIEW

File: examples/general/config-array/webpack.config.js [hunk 1]
```diff
@@ -47,11 +54,16 @@ module.exports = [
...
+        {
+          test: /\.(woff|woff2|eot|ttf|otf)$/,
+          use: 'url-loader?limit=10000&name=font/[hash].[ext]'
+        },
+        {
+          test: /\.svg$/,
+          use: 'url-loader?limit=8192&name=image/[hash].[ext]'
         }
       ]
-    },
```
Comment (evilebottnawi) [inline examples/general/config-array/webpack.config.js]: Please enable `minimize` for this example

**Comment by evilebottnawi** [comment]
Also please accept CLA

File: examples/general/config-array/webpack.config.js (hunk 1 above)
Comment (noscripter) [reply examples/general/config-array/webpack.config.js]: Sorry, my fault, please check again. I also remove bundle.js from layout.

**Comment by hiroppy** [comment]
@noscripter Could you rebase?

**Comment by noscripter** [comment]
> I think this repository should not have eot, less, svg and ttf of iconfont because this repository will become huge. Can you change to install via npm?

Sure, I'll modify it.

**Comment by noscripter** [comment]
> /cc @noscripter friendly ping

I've lost thread of this PR, maybe I'll close this for the moment until later I got more time.

**Pull request closed without merging**